tools/export/exporters.py

```diff
@@ -130,9 +130,13 @@ def make_key(self, src):
         Positional Arguments:
         src - the src's location
         """
-        key = basename(dirname(src))
-        if key == ".":
-            key = basename(realpath(self.export_dir))
+        rel_path = relpath(src, self.resources.file_basepath[src])
+        path_list = os.path.normpath(rel_path).split(os.sep)
+        assert path_list >= 1
```

Comment:
Thanks @c1728p9. Probably the easiest way to fix this is to just submit another PR, changing this assert to an exception that actually checks `len(path_list)`. @sarahmarshy feel free to submit this if you know the proper way of handling the exception!